tree/treeplayer/inc/ROOT/ParallelismUtils.hxx

```diff
@@ -0,0 +1,59 @@
+/*************************************************************************
+ * Copyright (C) 1995-2021, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+/**
+  \defgroup Parallelism Parallelism
+Functions and classes that allow parallel execution of ROOT code.
+*/
+
+/**
+ \file ROOT/ParallelismUtils.hxx
+ \ingroup Parallelism
+ \author Vincenzo Eduardo Padulano
```

Comment:
I feel the group name "Parallelism" may be too general/misleading (e.g. you check the Doxygen page and figure that ROOT does not have other functions or classes that allow parallel execution of ROOT code other than these). Also things like `GetTreeFullPaths` and `GetFriendInfo` do not really deal with parallelism. Maybe `TreeUtils`?

Also please add me and Enric as authors of this code :smile:

Reply:
Very sorry for the omission, my bad!

Reply:
About the naming, initially I called this file `TreePlayerUtils` and the namespace `ROOT::Internal::TreePlayer`, but then I switched to `Parallelism` since these functions are only used by TTreeProcessorMT which is in the doxygen group `Parallelism`. By the way, this doxygen group is not shown in the reference guide since there is no `\defgroup Parallelism`, so I added it to this new file. Though this is a separate issue and might very well go in another PR

Reply:
So should I rename the namespace to `ROOT::Internal::TreeUtils` ? Is it still fine to leave it `\ingroup Parallelism` after this?